db/sql/src/logs.rs

```diff
@@ -358,7 +362,12 @@ impl HoprDbLogOperations for HoprDb {
             .await?
             .perform(|tx| {
                 Box::pin(async move {
+                    // keep selected columns to a minimum to reduce copy overhead in db
                     let log_count = Log::find()
+                        .select_only()
+                        .column(log::Column::BlockNumber)
+                        .column(log::Column::TransactionIndex)
+                        .column(log::Column::LogIndex)
```

Comment:
Are all 3 needed? Shouldn't a single column be enough? Or does it need some sort of deduplication and demasking?

Reply:
I'd rather have all 3 since they compose the unique identifier. They are not needed though.